src/broker/overlay.c

```diff
@@ -1755,13 +1780,24 @@ static int overlay_configure_torpid (struct overlay *ov)
 
         if (flux_conf_unpack (flux_get_conf (ov->h),
                               &error,
-                              "{s?{s?s s?s}}",
+                              "{s?{s?s s?s s?i s?i s?i}}",
                               "tbon",
                                 "torpid_min", &min_fsd,
-                                "torpid_max", &max_fsd) < 0) {
+                                "torpid_max", &max_fsd,
+                                "keepalive_cnt", &ov->keepalive_cnt,
+                                "keepalive_idle", &ov->keepalive_idle,
+                                "keepalive_intvl", &ov->keepalive_intvl) < 0) {
```

Comment:
For config files, I'd argue against abbreviating these values unnecessarily (only saving 2 and 3 characters), i.e. `keepalive_count`, `keepalive_idle`, and `keepalive_interval`. However, that is just my opinion.

Reply:
Good idea, I was just using the zeromq names for these things, which are made up anyway.  Will do.

Reply:
Ah I didn't realize that you were using the zeromq names, and that those in turn were named after the sysctl settings. I kind of feel dumb for my suggestion now..

Reply:
Actually zeromq names are a little tweaked compared to the system names, e.g. system "probes" vs zmq "cnt", system "time" vs "idle".  I thought your suggestion was a good one.